ext/net/01_net.js

```diff
@@ -73,6 +78,7 @@
     }
 
     read(p) {
+      // TODO(bartlomieju): handle unrefing of connection
       return read(this.rid, p);
```

Comment:
Note to self: store if this conn is currently "unrefed" and if so immediately unref the promise returned from this API. Additionally all promise ids should be stored in an internal array so they all can be refed/unrefed when needed (and when the promise finishes the relevant promise id should be filtered out from that array).